src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -337,10 +337,13 @@ def add_view_to_instance(
             global_instance.instance_views.append(instance_view)
         self.local_id_to_global_id_map[env_id][local_instance_id] = global_instance_id
         if self.debug_visualize:
+            import os
+
```

Comment:
`os` gets imported locally so this shouldn't be necessary

Reply:
For some reason it is, @yvsriram any idea why?

Reply:
Not sure, checking now.

Reply:
Seems to work fine for me. I removed these. @cpaxton can you retry once again if you get a chance? But I think it should be fine.